setuptools/dist.py

```diff
@@ -289,8 +289,11 @@ def assert_bool(dist, attr, value):
         raise DistutilsSetupError(tmpl.format(attr=attr, value=value))
 
 
-def invalid(dist, attr, value):
-    raise DistutilsSetupError(f"{attr} is invalid.")
+def invalid_ignored_if_false(dist, attr, value):
+    if not value:
+        warnings.warn("{attr} is ignored")
```

Comment:
I think this should be
```suggestion
        warnings.warn("{attr} is ignored", DistDeprecationWarning)
```
(following the style in this module).